.travis.yml

```diff
@@ -38,14 +38,18 @@ before_install:
 
   - nvm install $NODE_VERSION
 
-  - if [ $TRAVIS_OS_NAME == "linux" ]; then
+  - if [ $TRAVIS_OS_NAME == "linux" ] && [ ! "$TRAVIS_TAG" ]; then
       export GYP_DEFINES="use_obsolete_asm=true";
       export JOBS=4;
       export CC=/usr/bin/gcc-4.9;
       export CXX=/usr/bin/g++-4.9;
+    else
+      export GYP_DEFINES="use_obsolete_asm=true";
```

Comment:
We default to that in https://github.com/nodegit/nodegit/blob/master/vendor/openssl/openssl.gyp#L12. That _might_ not be needed anymore?? /cc @implausible